app/models/validations/sales/sale_information_validations.rb

```diff
@@ -108,20 +108,20 @@ def validate_discount_and_value(record)
 
   def validate_non_staircasing_mortgage(record)
```

Comment:
The PR title makes me think this is the wrong method... I think there's a different validation for staircasing staircasing logs

Reply:
Ah, good point! I think it should be added to both. It's on hold atm tho, so will see what comes out of it!